src/Sepes.Infrastructure/Service/Azure/StorageAccount/AzureBlobStorageService.cs

```diff
@@ -39,7 +39,7 @@ public async Task<List<BlobStorageItemDto>> GetFileList(string containerName, Ca
                 while (await enumerator.MoveNextAsync())
                 {
                     var curBlob = enumerator.Current;
-                    result.Add(new BlobStorageItemDto() { Name = curBlob.Name, ContentType = curBlob.Properties.ContentType, Size = curBlob.Properties.ContentLength.HasValue ? curBlob.Properties.ContentLength.Value : 0 });
+                    result.Add(new BlobStorageItemDto() { Key = curBlob.Name, Name = curBlob.Name, ContentType = curBlob.Properties.ContentType, Size = curBlob.Properties.ContentLength.HasValue ? curBlob.Properties.ContentLength.Value : 0 , Modified = curBlob.Properties.LastModified.Value.ToUnixTimeMilliseconds()});
```

Comment:
Ref code scanning warning: maybe add a null check here for curBlob.Properties.LastModified, use curBlob.Properties.LastModified.HasValue